builtin/plugin/backend_test.go

```diff
@@ -1,4 +1,4 @@
-package plugin
+package plugin_test
```

Comment:
Any reason why this has to be broken out to its own package?

Reply:
Replied to this one in the above comment.